package.json

```diff
@@ -23,6 +23,7 @@
     "esdown": "latest",
     "espree": "latest",
     "esprima": "latest",
+    "fast-levenshtein": "^3.0.0",
```

Comment:
I pick this library because it's already in our n_m:
```
compat-table@0.0.0 /tmp/compat-table
└─┬ cheerio@0.20.0
  └─┬ jsdom@7.2.2
    └─┬ escodegen@1.14.3
      └─┬ optionator@0.8.3
        └── fast-levenshtein@2.0.6
```

Reply:
altho you picked v3, and v2's already what's there :-p

Reply:
Yeah. 🙈 My use-bleeding-edge progressive half eventually outweighs the other pick-one-from-node-modules conservative half.